osf/models/files.py

```diff
@@ -445,7 +445,7 @@ def update(self, revision, data, user=None):
                 break
         else:
             # Insert into history if there is no matching etag
-            if data.get('modified'):
+            if data.get('modified') and None not in [item['modified'] for item in self.history]:
```

Comment:
I understand the first part. But if the update() method is for the File class, what kind of "debug statement" should I add to the codebase? Are to log to sentry every time when the update() is called???